css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
```

Comment:
maybe i can cheat and make a rule that is more specific than the other. will see if i can do that as a quick fix/compromise.